src/mumble/UserModel.h

```diff
@@ -126,7 +127,7 @@ class UserModel : public QAbstractItemModel {
 		void setCommentHash(Channel *c, const QByteArray &hash);
 
 		void removeUser(ClientUser *p);
-		void removeChannel(Channel *c);
+		bool removeChannel(Channel *c, bool onlyIfUnoccupied = false);
```

Comment:
```suggestion
		bool removeChannel(Channel *c, const bool &onlyIfUnoccupied = false);
```

Reply:
Using a reference here doesn't make sense. I can add some const. Forgot I added const correctness to the coding guidelines way back ;)

Reply:
If you add `const`, using a reference is safe. In this case it's just a `bool` variable, meaning that in terms of performance there isn't a noticeable difference, but it's good practice.

Reply:
Added a const. Passing primitive variables as reference introduces an needless indirection (they fit a register anyways so why pass a pointer). Afaik best practice is to not use a reference for them.

Reply:
Interesting, I didn't know about that.